ydb/core/kqp/ut/view/view_ut.cpp

```diff
@@ -377,3 +385,82 @@ Y_UNIT_TEST_SUITE(TSelectFromViewTest) {
         }
     }
 }
+
+Y_UNIT_TEST_SUITE(TEvaluateExprInViewTest) {
+
+    Y_UNIT_TEST(EvaluateExpr) {
+        TKikimrRunner kikimr;
+        EnableViewsFeatureFlag(kikimr);
+        auto session = kikimr.GetTableClient().CreateSession().GetValueSync().GetSession();
+
+        constexpr const char* viewName = "TheView";
+        constexpr const char* timeQuery = R"(
+            SELECT EvaluateExpr(CurrentUtcTimestamp())
+        )";
+
+        const TString creationQuery = std::format(R"(
+                CREATE VIEW {} WITH (security_invoker = TRUE) AS {};
+            )",
+            viewName,
+            timeQuery
+        );
+        ExecuteDataDefinitionQuery(session, creationQuery);
+
+        const TString selectFromViewQuery = std::format(R"(
+                SELECT * FROM {};
+            )",
+            viewName
+        );
+        TExecDataQuerySettings queryExecutionSettings;
+        queryExecutionSettings.KeepInQueryCache(true);
+        const auto executeTwice = [&](const TString& query) {
+            return TVector<TString>{
+                GetYsonResults(session, query, queryExecutionSettings),
+                GetYsonResults(session, query, queryExecutionSettings)
+            };
+        };
+        const auto viewResults = executeTwice(selectFromViewQuery);
+        const auto etalonResults = executeTwice(timeQuery);
+        UNIT_ASSERT_EQUAL_C(viewResults[0] < viewResults[1], etalonResults[0] < etalonResults[1],
```

Comment:
This might be counter intuitive, but etalonResult[0] is equal to etalonResult[1] at the moment of writing this unit test. @gridnevvvit confirmed that this behaviour is somewhat expected.